components/dashboard/src/settings/settings-menu.ts

```diff
@@ -65,3 +47,40 @@ export default function getSettingsMenu(params: { showPaymentUI?: boolean; showU
         },
     ];
 }
+
+function renderBillingMenuEntries(billingMode?: BillingMode) {
+    if (!billingMode) {
+        return [];
+    }
+    switch (billingMode.mode) {
+        case "none":
+            return [];
+        case "chargebee":
+            return [
+                {
+                    title: "Plans",
+                    link: [settingsPathPlans],
+                },
+                {
+                    title: "Team Plans",
+                    link: [settingsPathTeams],
+                },
+            ];
+        case "usage-based":
+            return [
+                // We need to allow access to "Team Plans" here, at least for owners.
+                ...(billingMode.hasChargebeeTeamSubscription
+                    ? [
+                          {
+                              title: "Team Plans",
+                              link: [settingsPathTeams],
+                          },
+                      ]
+                    : []),
+                {
+                    title: "Billing",
+                    link: [settingsPathBilling],
+                },
```

Comment:
Why was this moved below `Team Plans`? To me it's more important, and should remain above it. 💭

Reply:
Addressed in https://github.com/gitpod-io/gitpod/pull/11814/commits/0006b8d855ca60773cf3fd1b2b9601e5f407fc32